src/plasma/plasma_manager.c

```diff
@@ -193,7 +194,7 @@ struct plasma_manager_state {
   client_connection *manager_connections;
   db_handle *db;
   /** Our address. */
-  uint8_t addr[4];
+  const char *addr;
```

Comment:
When you rebase, you will need to update the check I added in https://github.com/ray-project/ray/blob/master/src/plasma/plasma_manager.c#L836